### DISCUSSION TITLE
Context Menu for Mantine

### DISCUSSION THREAD

**Discussion by Nishchit14** [mantinedev/mantine#4439] [Feature requests]
I am developing an API platform called https://firecamp.dev/, and I'm currently working on a feature that involves allowing users to right-click on the API Collection name or the Request name to open a context menu with various options like renaming, exporting, duplicating, etc. While using the Mantine UI library, I couldn't find a built-in component specifically designed for context menus. It would be immensely helpful to have a **Context Menu** component within the Mantine library.

A notable example of context menus from radix-ui.
https://www.radix-ui.com/docs/primitives/components/context-menu

**Comment by rtivital** [comment]
You can use a third-party package for that – https://icflorescu.github.io/mantine-contextmenu/

**Reply by Nishchit14** [reply]
Looks promising. Great, thank you for sharing it. I wish it'll be built-in in the future. I really appreciate your work.  Thank you for bringing mantine to the real world.
